x/staking/exported/exported.go

```diff
@@ -0,0 +1,19 @@
+package exported
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+type (
+	// Subspace defines an interface that implements the legacy x/params Subspace
+	// type.
+	//
+	// NOTE: This is used solely for migration of x/params managed parameters.
+	Subspace interface {
```

Comment:
Why does this interface differ from what we did with the other modules? E.g. https://github.com/cosmos/cosmos-sdk/blob/main/x/mint/exported/exported.go#L8-L18